drs_client/client.py

```diff
@@ -0,0 +1,95 @@
+from drs_client.models import AccessURL, DrsObject, PostDrsObject
+from pydantic import BaseModel
+import requests
+import json
+
+
+class DRSClient():
+    """Client to communicate with a GA4GH DRS instance.
+    Supports additional endpoints defined in DRS-filer (add URL).
+
+    Args:
+        host: Base URL of DRS instance.
+        port: Port at which DRS instance can be accessed.
+        base-path: Path at which DRS endpoints can be accessed.
+        token: Bearer token for authentication
+    
+    Attributes:
+        url: URL to DRS endpoints, composed of `host`, `port` and `base_path`,
+            e.g.,"https://my-drs.app:8080/ga4gh/drs/v1"
+        token: Bearer token for authentication
+    """
+
+    def __init__(
+        self, 
+        host= 'http://0.0.0.0', 
+        port = '8080', 
+        base_path= 'ga4gh/drs/v1',
+        token = None
+    ) -> None:
+        self.url = f"{host}:{port}/{base_path}"
+        self.token = token
+
+    def get_object(self, object_id: str) -> str:
+        """ Gets the DRS object
+
+        Args:
+            object_id: Identifier of DRS object to be retrieved.
+
+        Returns:
+            DRS object as dictionary, JSONified if returned in app context.
+        """
+        request_url = f"{self.url}/objects/{object_id}"
+        req = requests.get(url=request_url)
+        if req:
+            DrsObject(**req.json())  # validate icoming payload
```

Comment:
Wouldn't it be better to return the unmarshalled response so that users can use dot notation to access the return objects' attributes rather than dictionary notation?
I also don't understand the logic (`if req:`): if it's _not_ true, how can you access `req.json()` in the call after the conditional? In fact, I think I would probably just do the following:

```py
return DrsObject(**req.json())
```

or better:

```
try:
    return DrsObject(**req.json())
except Exception as e:
   # log exception
   # access HTTP status code from `req` and return that instead